tutorial/airfoilopt/multipoint/airfoil_multiopt.py

```diff
@@ -36,8 +36,10 @@
 MP.addProcessorSet("cruise", nMembers=nGroup, memberSizes=nProcPerGroup)
 comm, setComm, setFlags, groupFlags, ptID = MP.createCommunicators()
 outputDirectory = "output"
-if comm.rank == 0 and os.path.exists(outputDirectory) is False:
+if not os.path.exists(outputDirectory):
     os.mkdir(outputDirectory)
+else:
+    raise Warning("The directory already exists! Please delete it or provide a new path")
```

Comment:
Should be an error so the script exits, maybe `OSError`?

Reply:
Warning exits as well, at least on my machine! Happy to change to `OSError` for consistency though!

Reply:
Ah right, in Python the `Warning` class is derived from the `Exception` class, so by default they will raise an error. However this is less explicit and we should raise a specific error, in this case `OSError` is probably the most appropriate. I'd prefer to leave warnings for things like `DeprecationWarning` etc.